pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -2560,6 +2560,21 @@ func (gce *GCECloud) GetAutoLabelsForPD(name string, zone string) (map[string]st
 	return labels, nil
 }
 
+// TestDisk checks that a disk has given type. It should be used only for
+// testing!
+func (gce *GCECloud) TestDisk(diskName, volumeType string) error {
```

Comment:
Why is this part of the cloud provider?  I don't like "only for testing" functions in the API.  I'd rather make `getDiskByNameUnknownZone` a public interface.  Is that doable?